tests/test_backoff_integration.py

```
@@ -5,100 +5,88 @@
 if TYPE_CHECKING:
-    from collections.abc import Generator
-
     from chaturbate_poller.core.client import ChaturbateClient
 
 import pytest
 from httpx import Request, Response
 
-from chaturbate_poller.config.backoff import BackoffConfig, backoff_config
-from chaturbate_poller.exceptions import PollingError
+from chaturbate_poller.config.backoff import BackoffConfig
+from chaturbate_poller.core.client import ChaturbateClient
```

Flake8 (7.2.0)

[error] redefinition of unused 'ChaturbateClient' from line 8

(F811)